pkg/monitor/dynkeepalived.go

```diff
@@ -163,7 +163,7 @@ func handleBootstrapStopKeepalived(kubeconfigPath string, bootstrapStopKeepalive
 	}
 
 	for {
-		if _, err := config.GetIngressConfig(kubeconfigPath, []string{}); err != nil {
+		if _, err := config.GetIngressConfig(kubeClient, []string{}); err != nil {
```

Comment:
It seems like this never fails now so we never stop keepalived on the bootstrap. Since we don't actually care about the ingress config here we could probably just swap this to a simple http.Get like below.